packages/console/src/assets/docs/tutorial/integrate-sdk/go-web.mdx

```diff
@@ -0,0 +1,386 @@
+
+<Alert>
+  We do NOT recommend using cookie-based sessions, as user authentication information stored by Logto may exceed the cookie size limit.
+  In this example, we use memory-based sessions. You can use Redis, MongoDB, and other technologies in production to store sessions as needed.
+</Alert>
+
+The `Storage` type in the Logto SDK is as follows:
+
+```go
+// github.com/logto-io/client/storage.go
+package client
+
+type Storage interface {
+	GetItem(key string) string
+	SetItem(key, value string)
+}
+```
+
+We will use [github.com/gin-contrib/sessions](https://github.com/gin-contrib/sessions) as an example to demonstrate this process.
+
+### Apply session middleware
+
+Apply the [github.com/gin-contrib/sessions](https://github.com/gin-contrib/sessions) middleware to the application, so that we can get the user session by the user request context in the route handler:
+
+
+```go
+package main
+
+import (
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-contrib/sessions/memstore"
+	"github.com/gin-gonic/gin"
+	"github.com/logto-io/go/client"
+)
+
+func main() {
+	router := gin.Default()
+
+	// We use memory-based session in this example
+	store := memstore.NewStore([]byte("your session secret"))
+	router.Use(sessions.Sessions("logto-session", store))
+
+	router.GET("/", func(ctx *gin.Context) {
+		// Get user session
+		session := sessions.Default(ctx)
+		// ...
+		ctx.String(200, "Hello Logto!")
+	})
+	router.Run(":8080")
+}
+```
+
+### Create session storage for Logto to store user authentication information
+
+Create a `session_storage.go` file, define a `SessionStorage` and implement the Logto SDK's `Storage` interfaces:
+
+```go
+// session_storage.go
+package main
+
+import (
+	"github.com/gin-contrib/sessions"
+)
+
+type SessionStorage struct {
+	session sessions.Session
+}
+
+func (storage *SessionStorage) GetItem(key string) string {
+	value := storage.session.Get(key)
+	if value == nil {
+		return ""
+	}
+	return value.(string)
+}
+
+func (storage *SessionStorage) SetItem(key, value string) {
+	storage.session.Set(key, value)
+	storage.session.Save()
+}
+```
+
+Now, in the route handler, you can create a session storage for Logto as follows:
+
+```go
+session := sessions.Default(ctx)
+sessionStorage := &SessionStorage{session: session}
+```
+
+</Step>
+
+<Step
+  title="Init LogtoClient"
+  subtitle="2 steps"
+  index={2}
+  activeIndex={props.activeStepIndex}
+  onButtonClick={() => props.onNext(3)}
+>
+
+### Create LogtConfig
+
+<pre>
+<code className="language-go">
+{`// main.go
+func main() {
+	// ...
+
+	logtoConfig := &client.LogtoConfig{
+		Endpoint:           "${props.endpoint}",
+		AppId:              "${props.appId}",
+		AppSecret:          "${props.appSecret}",
+		PersistAccessToken: true,
+	}
+
+	// ...
+}`}
+</code>
+</pre>
+
+The `PersistAccessToken` config indicates whether the access token needs to be stored in the session.
+
```

Comment:
We can add some descriptions on why it is set to `true`.

Reply:
Per offline discussion, we don't need to explain why this needs to be `true`. It's straightforward.
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1662445224089459